docs/topics/testing/overview.txt

```diff
@@ -257,6 +257,14 @@ To prevent serialized data from being loaded twice, setting
 :data:`~django.db.models.signals.post_migrate` signal when flushing the test
 database.
 
+.. versionchanged:: 2.2
+
+    Data are now loaded at the end of a ``TransactionTestCase``, after
```

Comment:
In this case it can be omitted then. I don't see anything from what I added there that should remain afterwards. I guess the release note explaining this behavior change is enough.